src/index.ts

```diff
@@ -65,9 +66,9 @@ function activateHubExtension(app: JupyterLab, palette: ICommandPalette, mainMen
 
   commands.addCommand(CommandIDs.logout, {
     label: 'Logout',
-    caption: 'Log out of the Hub',
+    caption: 'Shut down this server and log out of the Hub',
```

Comment:
My understanding was that the issue in #51 was that the server was still accessible specifically because it was still running.  I don't know if there is a way to leave the server running while also logging out in the browser.

As far as I can see, though, this change uses the same logout URL as the classic interface ('/tree/'), so it should behave the same way.

Reply:
Ok, in that case we should keep the menu item label as "Log out...". Visiting the old link does not stop the pod.

Reply:
Ah, okay.  Done.